onnxruntime/core/framework/debug_node_inputs_outputs_utils.cc

```diff
@@ -202,6 +202,16 @@ const NodeDumpOptions& NodeDumpOptionsFromEnvironmentVariables() {
       opts.data_destination = NodeDumpOptions::DataDestination::TensorProtoFiles;
     }
 
+    if (get_bool_env_var(env_vars::kRankToFileName)) {
+      char const* tmp = Env::GetEnvironmentVar("OMPI_COMM_WORLD_RANK");
```

Comment:
does this compile? would expect it to be `Env::Default().GetEnvironmentVar("...")` which returns a std::string